trade-finance-manager-api/src/graphql/reducers/mappings/gef-facilities/mapGefFacilityDates.api-test.js

```diff
@@ -72,7 +72,7 @@ describe('mapGefFacilityDates', () => {
     expect(result.tenor).toEqual(expected);
   });
 
-  it('should return object with undefined coverEndDate if coverEndDate is null', () => {
+  it('could just say should not map coverEndDate when date is null', () => {
```

Comment:
I'm not sure I understand the title of this test?

Reply:
I dunno how I managed to do that 😂

Reply:
done